src/NServiceBus.Core/Routing/IRouteSource.cs

```diff
@@ -0,0 +1,11 @@
+namespace NServiceBus
+{
+    using System;
+    using Routing;
+
+    interface IRouteSource
+    {
+        void GenerateRoutes(Action<RouteTableEntry> registerRouteCallback);
+        RouteSourcePriority Priority { get; }
```

Comment:
I'd keep that as is because it is internal stuff and we can change it whenever we want.